bouncer/views.py

```diff
@@ -60,6 +60,11 @@ def annotation(self):
         extension_url = "{uri}#annotations:{id}".format(
             uri=document_uri, id=annotation_id)
 
+        parsed_url = parse.urlparse(document_uri)
+        parsed_path = parsed_url.path
+        if parsed_path.endswith("/"): parsed_path = parsed_path[:-1]
+        pretty_url = parsed_url.netloc + parsed_path
```

Comment:
Do you want to add `parsed_path` on here, actually? This produces "Loading annotation for www.seanh.cc/posts/functional-core-imperative-shell" for example. Do we just want "Loading annotation for www.seanh.cc" instead?